spec/factories/schedulable_cutoff_dates.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :schedulable_cutoff_date do
+    id { 1 }
```

Comment:
```suggestion
    sequence(:id)
```
This will prevent a pkey collision whenever running the factory multiple times.

Reply:
I've opted to just get rid of the id from the factory, the pk should automatically be sequenced